### ISSUE TITLE
redirect won't go to `/` but empty quotes works

### ISSUE THREAD

**Issue by arxpoetica** [sveltejs/sapper#291]
(In preload) like so:

```js
return this.redirect(302, '/') // it be broke
return this.redirect(302, '') // it work
```

**Comment by arxpoetica** [comment]
Probably another way to say this, leading slashes don't work.

**Comment by Rich-Harris** [comment]
How do you think this ought to be handled? Should Sapper just strip any leading `/` character?

**Comment by arxpoetica** [comment]
Good Q. Maybe follow Express conventions? From:

> Express uses [path-to-regexp](https://www.npmjs.com/package/path-to-regexp) for matching the route paths; see the [path-to-regexp documentation](http://forbeslindesay.github.io/express-route-tester/) for all the possibilities in defining route paths.

It appears the leading `/` is optional: `/^\/?$/i`

**Comment by s3ththompson** [comment]
This threw me today. I expected the routes to be relative if they didn't have a starting `/`...

**Comment by Rich-Harris** [comment]
released 0.15.7 with the fix
